src/DependencyInjection/ServiceContainer.php

```diff
@@ -76,6 +76,6 @@ private function initContainer()
         );
         $containerProvider = new ContainerProvider($this->moduleName, $this->moduleLocalPath, $cacheDirectory);
 
-        $this->container = $containerProvider->get(defined('_PS_ADMIN_DIR_') || defined('PS_INSTALLATION_IN_PROGRESS') ? 'admin' : 'front');
+        $this->container = $containerProvider->get(defined('_PS_ADMIN_DIR_') || defined('PS_INSTALLATION_IN_PROGRESS') || (PHP_SAPI === 'cli') ? 'admin' : 'front');
```

Comment:
Are the parenthesis mandatory?

Reply:
no, but it's more readable ?!
I will delete them